### PR TITLE
Test for volume create command

### PR DESCRIPTION
**Pull request by suyanhanx**
Signed-off-by: Han Xu <suyanhanx@gmail.com>

### CODE REVIEW

File: cmd/nerdctl/volume_create_test.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+import (
+	"testing"
+
+	"github.com/containerd/nerdctl/pkg/testutil"
+	"gotest.tools/v3/assert"
+)
+
+// Test TestVolumeCreate for creating volume with given name.
+func TestVolumeCreate(t *testing.T) {
```
Comment (djdongjin) [inline cmd/nerdctl/volume_create_test.go:27]: `TestVolumeCreate` and `TestVolumeCreateWithLabels` seems duplicated with tests in `volume inspect`

https://github.com/containerd/nerdctl/blob/b80c9caa28c849c5191e834dd072c11ff80ee9b1/cmd/nerdctl/volume_inspect_test.go#L29

Comment (suyanhanx) [reply cmd/nerdctl/volume_create_test.go:27]: Yes. I saw it. But it feels like this should be moved from the inspect side to the create side ?

Comment (fahedouch) [reply cmd/nerdctl/volume_create_test.go:27]: As integration test, it is ok to keep both

File: cmd/nerdctl/volume_create_test.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	base := testutil.NewBase(t)
+	testVolume := testutil.Identifier(t)
+
+	base.Cmd("volume", "create", testVolume).AssertOK()
+	defer base.Cmd("volume", "rm", testVolume).Run()
```
Comment (fahedouch) [inline cmd/nerdctl/volume_create_test.go:32]: `defer base.Cmd("volume", "rm", "-f", testVolume).Run()`

File: cmd/nerdctl/volume_create_test.go (hunk 1 above)
Comment (djdongjin) [reply cmd/nerdctl/volume_create_test.go:27]: SGTM

File: cmd/nerdctl/volume_create_test.go (hunk 2 above)
Comment (suyanhanx) [reply cmd/nerdctl/volume_create_test.go:32]: Make sense.

**Review by fahedouch — APPROVED** [review]
Thanks

**Pull request merged**